activations/activationHeight.go

```diff
@@ -61,61 +62,59 @@ func init() {
 		panic("ACTIVATION_TYPE_COUNT does not match the list of Activations")
 	}
 
-	ActivationMap = make(map[ActivationType]Activation, len(activations))
-	ActivationNameMap = make(map[ActivationType]string, len(activations))
+	activationMap = make(map[ActivationType]activation, len(activations))
+	activationNameMap = make(map[ActivationType]string, len(activations))
 	for _, a := range activations {
-		ActivationMap[a.Id] = a
-		ActivationNameMap[a.Id] = a.Name
+		activationMap[a.id] = a
+		activationNameMap[a.id] = a.name
 	}
 }
 
 // String converts an Activation ID to a name
 func (id ActivationType) String() string {
 
-	n, ok := ActivationNameMap[id]
+	n, ok := activationNameMap[id]
 	if !ok {
 		//n = fmt.Sprintf("ActivationId(%v)", id)
```

Comment:
would you mind fixing this while you are at it 👀

Reply:
Done. If you meant something else than what I did, please let me know